examples/e2e/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+services:
```

Comment:
I needed `version: "3"` at the start for my local `docker-compose` (1.25.0) to parse this